tools/docker/static-analysis/cppcheck.sh

```diff
@@ -12,7 +12,6 @@ INCLUDES="\
     -I$rootdir/framework/common/include \
     -I$rootdir/framework/tlvf/src/include \
     -I$rootdir/framework/tlvf/AutoGenerated/include \
-    -I$rootdir/framework/discovery_agent/include \
```

Comment:
Can we please please delete the discovery agent while we're at it? Its not used and will not be used.